### DISCUSSION TITLE
Not seeing some metrics in a Dashboard.

### DISCUSSION THREAD

**Discussion by tvaril** [plausible/analytics#2973] [Self-Hosted Support]
Hi, I noticed that I am missing some metric in Dashboard vs what I can see on Plausible website and Docs. Is it a difference between paid version and self-hosted version? Or am I doing something wrong or missed some configuration when deployed plausible to Docker? From the [documentation](https://plausible.io/docs/guided-tour) I read that I should be seeing this:

> See the number of unique visitors, total visits (sessions), total page views, views per visit, the bounce rate (the percentage of sessions that had one page view only) and the visit duration.

But I am seeing only: Unique visitors, total pageviews, bounce rate and visit duration.

Many thanks in advance

**Comment by ruslandoga** [comment]
👋 @myazaki 

https://github.com/plausible/analytics/discussions/2896#discussioncomment-5801977 might be relevant.

**Reply by tvaril** [reply]
Thanks that was it :-) sorry for duplication...
